src/webgl/framebuffer.js

```diff
@@ -187,15 +186,6 @@ export default class Framebuffer extends Resource {
         this._attachTexture({attachment, texture: object, layer: 0, level: 0});
       }
 
-      if (setFilteringParameters && object instanceof Texture2D) {
```

Comment:
OK - They should only play a role when rendering to textures. I have had problems with this in the past but perhaps unrelated.